spec/pagseguro/transaction_request_spec.rb

```diff
@@ -120,8 +120,7 @@
       let(:raw_xml) { File.read("./spec/fixtures/invalid_code.xml") }
 
       it "does not create a transaction request" do
-        expect(transaction_request.create).to be_falsy
-        expect(transaction_request.code).to be_nil
+        expect { transaction_request.create }.not_to change { transaction_request.code }
```

Comment:
[`PagSeguro::TransactionRequest#create`](https://github.com/pagseguro/ruby/blob/2.5.0-dev/lib/pagseguro/transaction_request.rb#L101) calls `Request` that calls [`PagSeguro::TransactionRequest::Response#serialize`](https://github.com/pagseguro/ruby/blob/2.5.0-dev/lib/pagseguro/transaction_request/response.rb) to serialize the response from server, the response can be:

* successful: this will update attributes that only becomes from server (like `code`);
* error: this will add errors to object;

So you can think like in rails: how do you know that an `ActiveRecord` object is stored to database? => because it is with no errors and the object has a `id`.

P. S.: Maybe we can add a check by error, but a think this one is a little overkill :/